Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN python manage.py collectstatic --noinput &&\
 	python manage.py compress
 
 RUN printf '#!/bin/bash\nxvfb-run -a --server-args="-screen 0, 1024x768x24" /usr/bin/wkhtmltopdf -q $*' > /usr/local/bin/wkhtmltopdf.sh \
-  && chmod +x /usr/local/bin/wkhtmltopdf.sh
+  && chmod +x /usr/local/bin/wkhtmltopdf.sh &&\
+  ln -s /usr/local/bin/wkhtmltopdf.sh /usr/local/bin/wkhtmltopdf
```

Comment:
small niggle but please put the && at the start of this line rather than at the end of the previous line.
It makes it more consistent and neater

Reply:
I updated it @KaitCrawford 
Thanks for the check